.github/workflows/rpm-build-and-test.yml

```diff
@@ -231,8 +231,8 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
         with:
-          submodules: true
-          fetch-depth: 500
+          submodules: false
+          fetch-depth: 1
```

Comment:
This is the default isn't it?

Reply:
If the defaults change then the version number changes, it was changing from 2 to 3 that made the default fetch-depth 1.

Reply:
Updated.